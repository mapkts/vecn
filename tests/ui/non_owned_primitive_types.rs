#[vecn::vector]
struct TupleStruct(&f32 , &f32);

#[vecn::vector]
struct NamedStruct {
    x: &f32,
    y: &f32,
}

fn main() {}
