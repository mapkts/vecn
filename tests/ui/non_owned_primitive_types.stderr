error: expected owned primitive type, but found others
 --> $DIR/non_owned_primitive_types.rs:2:19
  |
2 | struct TupleStruct(&f32 , &f32);
  |                   ^^^^^^^^^^^^^

error: expected owned primitive type, but found others
 --> $DIR/non_owned_primitive_types.rs:5:20
  |
5 |   struct NamedStruct {
  |  ____________________^
6 | |     x: &f32,
7 | |     y: &f32,
8 | | }
  | |_^
